src/Sparrow/LowMemory/LowMemoryNotification.cs

```diff
@@ -44,10 +44,24 @@ internal class LowMemEventDetails
         private int _clearInactiveHandlersCounter;
         private bool _wasLowMemory;
 
-        private void RunLowMemoryHandlers(bool isLowMemory, LowMemorySeverity lowMemorySeverity = LowMemorySeverity.ExtremelyLow)
+        private void RunLowMemoryHandlers(bool isLowMemory, MemoryInfoResult memoryInfo, LowMemorySeverity lowMemorySeverity = LowMemorySeverity.ExtremelyLow)
         {
             try
             {
+                try
+                {
+                    if (_logger.IsOperationsEnabled)
+                        _logger.Operations($"Running {_lowMemoryHandlers.Count} low memory handlers with severity: {lowMemorySeverity}. " +
+                                           $"Commit charge: {memoryInfo.CurrentCommitCharge} / {memoryInfo.TotalCommittableMemory}, " +
+                                           $"Memory: {memoryInfo.TotalPhysicalMemory - memoryInfo.AvailableMemory} / {memoryInfo.TotalPhysicalMemory}, " +
+                                           $"Managed memory: {new Size(AbstractLowMemoryMonitor.GetUnmanagedAllocationsInBytes(), SizeUnit.Bytes)}, " +
+                                           $"Unmanaged allocations: {new Size(AbstractLowMemoryMonitor.GetUnmanagedAllocationsInBytes(), SizeUnit.Bytes)}");
+                }
+                catch
+                {
+                    // ignored
```

Comment:
explain that this can fail due to OOM error here

Reply:
done